code/game/machinery/doors/airlock.dm

```diff
@@ -1018,10 +1018,10 @@
 		return
 
 	if(!operating)
-		if(istype(I, /obj/item/twohanded/fireaxe)) //being fireaxe'd
-			var/obj/item/twohanded/fireaxe/F = I
-			if(!F.wielded)
-				to_chat(user, "<span class='warning'>You need to be wielding the fire axe to do that!</span>")
+		if(istype(I, /obj/item/fireaxe)) //being fireaxe'd
+			var/obj/item/fireaxe/axe = I
+			if(!SEND_SIGNAL(axe, COMSIG_IS_TWOHANDED_WIELDED))
```

Comment:
Same concept of change as the bostaff needs to be applied here as well

Reply:
This was resolved now